frontend/src/Components/UserProfileName.tsx

```diff
@@ -15,6 +16,7 @@ export default function UserProfileName(props: UserProfileNameProps) {
   const [editing, setEditing] = useState(false);
   const [name, setName] = useState(props.name);
   const api = useAPI();
+  const refEditName = useHotkeys<HTMLInputElement>('enter', () => handleEditNameComplete(), {enableOnFormTags: ['INPUT'], preventDefault: true});
```

Comment:
we don't care much about it but I don't think we should skip on this - this is not accessible - instead we should have a wrapping `form` element and then clicking enter or using voice browser command will submit. In addition phone browser will recognize that they are submitting the form and will replace "enter" on virtual keyboard with "submit"

Reply:
Yeah, also had similar thoughts. Ideally we should have a reusable component that handles that.